Sources/TuistKit/ProjectEditor/ProjectEditorMapper.swift

```diff
@@ -54,12 +54,19 @@ final class ProjectEditorMapper: ProjectEditorMapping {
         targets.append(manifestsTarget)
         if let helpersTarget = helpersTarget { targets.append(helpersTarget) }
 
+        // Run Scheme
+        let buildAction = BuildAction(targets: targets.map { TargetReference(projectPath: sourceRootPath, name: $0.name) })
+        let arguments = Arguments(launch: ["generate --path \(sourceRootPath)": true])
+        let runAction = RunAction(configurationName: "Debug", filePath: "/usr/local/bin/tuist", arguments: arguments)
```

Comment:
Is this the correct path of `tuist`? Will it work with custom pinned version?

Reply:
We can't rely on this always being the path, especially since we are not far off having support for other operating systems - we can probably use `$(which tuist)` to locate it since that is available on any operating system with GNU.

Reply:
Agree with @ollieatkinson here. I would not hardcode an absolute path. Does it work if you just use `tuist`?

Reply:
> Does it work if you just use tuist?

Sadly for this it needs to be an absolute path.

Another option you can employ is consulting launch arguments - the first argument is the path to the binary in use. 

```swift
private func obtainCurrentExecutablePath() -> AbsolutePath {
    return AbsolutePath(ProcessInfo.processInfo.arguments[0])
}
```

Seeing it's a temporary project it's safe to use that - it will be to same binary used to invoke the `edit` command - that way you can get it working locally when you do `swift run` during development as well using the distributed `tuist`.

Reply:
@kwridan @ollieatkinson updated. Now tuist path that was used to launch the process gets injected here

Reply:
Excellent solution 👌🏻